config/400-config-istio.yaml

```diff
@@ -56,10 +83,44 @@ data:
     # sidecar injection (like Knative's istio-ci-no-mesh.yaml).  Since every pod
     # is outside of the service mesh in that case, a cluster-local  service
     # will need to be exposed to a cluster-local gateway to be accessible.
+    #
+    ##############
+    # NEW FORMAT #
+    ##############
```

Comment:
Seems a bit verbose. I think it is fine to just point from the old key to the new one, e.g. as in https://github.com/knative/networking/blob/main/config/config-network.yaml#L117. We don't need to say deprecated, but something like you have above "should be preferred.".